src/gui/widgets/spinbox.cpp

```diff
@@ -0,0 +1,180 @@
+/*
+	Copyright (C) 2008 - 2024
+	by babaissarkar(Subhraman Sarkar) <suvrax@gmail.com>
```

Comment:
No argument that you have a copyright on it, but when the previous line says "2008 -" it probably should have a second person too.

Reply:
Or more likely, just change the start year to 2023.

Reply:
Updated the start year to 2023.